docker/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM openjdk:8-jre-alpine
+
+ENV APP_PORT 8086
```

Comment:
This is the default port now, right? It can also be updated in the README.md then.

Reply:
Hm, apparently that's not what server.port in application.yml or application-default.yml does.